Refactor all the tests.

phoeinx opened the pull request.

phoeinx committed:
Refactor all the tests.

phoeinx committed:
added class comments and followed the path of the swalint

phoeinx committed:
Change credentials of test account

leowe commented on packages/TogglClient-Core.package/TogglAccountSettings.class/instance/buildWith..st:
```diff
@@ -5,7 +5,7 @@ buildWith: builder
 		label: #labelString;
 		autoCancel: true;
-		extent: 200 @ 60;
+		extent: self defaultExtent;
```
is that used somewhere else? otherwise it looks like unnecessary complexity

phoeinx replied:
das heißt wir bleiben bei defaultExtent?

leowe commented on packages/TogglClient-Core.package/TogglClient.class/README.md:
```diff
@@ -0,0 +1 @@
+Provides UI and logic for displaying a Toggl client in Squeak
```
"displaying a Toggl client" - what does that mean? how do you display a client? also Toggl is written with a capital t whereas in the other comments it is written with a lowercase t

ma-ro replied:
feel free to add alternative class comment

leowe commented on packages/TogglClient-Core.package/TogglClient.class/instance/actionLoginLogout.st:
```diff
@@ -2,12 +2,6 @@ actions
-				changed: #timeEntryList;
-				changed: #authenticated]
-		ifFalse: [TogglAccountSettings openWithToggl: self andAccount: self account].
+		ifTrue: [self logout]
```
logout is not the only thing that happens there, more descriptive title would be good. logoutAndCleanTimeEntryList or something like this

ma-ro replied:
done

leowe commented on packages/TogglClient-Core.package/TogglClient.class/instance/authenticated..st:
```diff
@@ -1,4 +1,4 @@
-initialisation
+initialize-release
```
wrong category, should be accessing

leowe commented on packages/TogglClient-Core.package/TogglClient.class/instance/authenticated.st:
```diff
@@ -1,4 +1,4 @@
-initialisation
+initialize-release
```
same

leowe commented on packages/TogglClient-Core.package/TogglClient.class/instance/defaultExtent.st:
```diff
@@ -0,0 +1,4 @@
+toolbuilder
```
inconsistent categorisation with other default extent + magic number encapsulation

leowe commented on packages/TogglClient-Core.package/TogglClient.class/instance/buildWith..st:
```diff
@@ -4,7 +4,7 @@ buildWith: aBuilder
 	^ aBuilder build: (aBuilder pluggableWindowSpec new
 		model: self;
 		label: #clientLabel;
-		extent: 500 @ 400;
```
still not a fan

leowe commented on packages/TogglClient-Core.package/TogglClient.class/instance/logout.st:
```diff
@@ -0,0 +1,9 @@
+toolbuilder
```
wrong category

leowe replied:
naming problem still holds

leowe commented on packages/TogglClient-Core.package/TogglClient.class/instance/wantsSteps.st:
```diff
@@ -1,4 +1,4 @@
-actions
+testing
```
seems like the wrong category

leowe commented on packages/TogglClient-Core.package/TogglJsonParser.class/README.md:
```diff
@@ -0,0 +1 @@
+Converts json strings to TogglTimeEntries and back
```
... and the other way around

leowe commented on packages/TogglClient-Core.package/TogglTimeEntry.class/README.md:
```diff
@@ -0,0 +1 @@
+Represents a time tracking entry in toggl.
```
first time a dot at the end

leowe commented on packages/TogglClient-Core.package/TogglTimeEntry.class/instance/renameTo..st:
```diff
@@ -1,4 +1,4 @@
-rename
+testing
```
testing seems really wrong

leowe commented on packages/TogglClient-Tests.package/TogglAPITests.class/instance/testAuthenticationSucceedsWithValidAccount.st:
```diff
@@ -0,0 +1,9 @@
+	response := client togglAPI
+		authenticateWithUsername: self validAccountUser
+		Password: self validAccountPassword.	
+	self assert: response isSuccess.
```
dot at the end

leowe commented on packages/TogglClient-Tests.package/TogglAPITests.class/instance/testPushLocalTimeEntriesSychronizesLocalTimeEntries.st:
```diff
@@ -0,0 +1,16 @@
+
+	| account |
+	account := TogglAccount new 
+				userName: self validAccountUser; 
```
indentation seems off

leowe commented on packages/TogglClient-Tests.package/TogglTimeEntryTests.class/instance/testTimeEntrySmallerThanAnotherTimeEntry.st:
```diff
@@ -0,0 +1,4 @@
+tests
+testTimeEntrySmallerThanAnotherTimeEntry
+
+	self assert: finishedTimeEntry < startedTimeEntry.
```
dot at the end

leowe commented on packages/TogglClient-Tests.package/TogglTimeEntryTests.class/instance/testTimeEntrySmallerThanOrEqualThanAnotherTimeEntry.st:
```diff
@@ -0,0 +1,4 @@
+tests
+testTimeEntrySmallerThanOrEqualThanAnotherTimeEntry
+
+	self assert: finishedTimeEntry <= startedTimeEntry.
```
dot at the end

ma-ro committed:
resolved review comments
	renamed caterogies
	renamed class comments
	made methods dot free
	renamed logout method

ma-ro committed:
deleted two more empty caterogies

phoeinx committed:
Refactor '' to String empty.

leowe committed:
Update README.md
